docs/builds.rst

```diff
@@ -84,17 +85,6 @@ For MkDocs, we run ``mkdocs build``.
 Then these files are copied across to our application servers from the build server.
 Once on the application servers, they are served from nginx. 
 
-An example in code:
-
-.. code-block:: python
-
-    update_docs_from_vcs(version)
-    if exists('setup.py'):
-        run('python setup.py install')
-    if project.requirements_file:
-        run('pip install -r %s' % project.requirements_file)
-    build_docs(version=version)
-    copy_files(artifact_dir)
```

Comment:
I wonder if we should still include some updated version of this. I think it's useful for users to get a mental model of what's happening.

Reply:
I thought about this, but I came up that it was more complicated to explain in code now considering that we have many options on our config file. In the end, I seemed to me to be more confusing than clarifying. 

What about this? I tried to keep some consistency with YAML options.

```python
    update_docs_from_vcs(version)
    config = get_project_config(project)
    if config.python.install.method.setuptools:
        run('python setup.py install')
    if config.python.install.method.pip:
        run('pip install .')
    if config.python.install.requirement:
        run('pip install -r %s' % config.python.install.requirement)
    build_docs(version=version)
    copy_files(artifact_dir)
```

It's not 100% accurate, but maybe it's better than removing it. What do you think?

Reply:
Looks good to me. :+1: